src/audio/pipeline/pipeline-graph.c

```diff
@@ -123,13 +122,12 @@ struct pipeline *pipeline_new(struct sof_ipc_pipe_new *pipe_desc,
 
 	/* init pipeline */
 	p->sched_comp = cd;
+	p->comp_id = comp_id;
+	p->pipeline_id = pipeline_id;
```

Comment:
no need to set `.priority`?

Reply:
your right, and it's not been used today. Lets keep it as we will use it in the future. Now fixed.